content/kubermatic/master/installation/install_kkp_CE/add_seed_cluster_CE/_index.en.md

```diff
@@ -210,10 +234,6 @@ When your cloud provider supports LoadBalancers, you can find the target IP / ho
 
 ```bash
 kubectl -n kubermatic get services
-```
-
-Output will be similar to this:
-```bash
```

Comment:
This was intentionally split into two code blocks, so it’s easy to copy the command. I would leave it as it is.

Reply:
We don't do this anywhere else in the documentation. We always have the output example right in the same code block.

Reply:
Just because we don't do this anywhere doesn't mean it's the right thing to do. :slightly_smiling_face: 

Having both input and output in the same block renders the copy functionality useless, but that functionality can be quite useful for readers.